modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,488 @@
+= Configure a Customer-Managed VPC on GCP
+:description: Connect Redpanda Cloud to your existing VPC resources for additional security.
+:page-cloud: true
+
+You can connect Redpanda Cloud to your customer-managed virtual private cloud (VPC), and you can then create clusters with the most security. With standard BYOC clusters using a Redpanda-managed VPC, Redpanda creates a VPC in your Google Cloud account and manages its resources. With a customer-managed VPC:
+
+* You provide your own VPC in your Google Cloud account.
+* You maintain more control of your Google Cloud account, because Redpanda requires less permissions than standard BYOC clusters.
+* You control your security resources and policies, including subnets, service accounts, IAM roles, firewall rules, and storage buckets.
+
+[NOTE]
+====
+include::shared:partial$feature-flag.adoc[]
+====
+
+== Prerequisites
+
+* Redpanda creates a private Google Kubernetes Engine (GKE) cluster in the VPC. For more information, see the https://cloud.google.com/kubernetes-engine/docs/how-to/service-accounts[GKE service account documentation^].
+* A standalone GCP project is recommended. If your host project (where your VPC project is created) and your service project (where your Redpanda cluster is created) are in different projects, you must first provision a shared VPC in Google Cloud. For more information, see the https://cloud.google.com/vpc/docs/provisioning-shared-vpc[Google shared VPC documentation^]. 
+* The subnet and secondary IP ranges you provide must allow public internet access. The configuration requires you to provide reserved CIDR ranges for the subnet and GKE Pods, Services, and master IP addresses. See <<Configure your VPC>>.
+* Redpanda requires access to certain Google APIs, storage buckets, and service accounts. See <<Configure the service project>>.
+
+== Limitations
+
+* Existing clusters cannot be moved to a customer-managed VPC.
+* After creating a cluster with a customer-managed VPC, you cannot change the VPC.
```

Comment:
You cannot change to another VPC? Or you cannot edit the VPC?

Reply:
fixed to you cannot change to a different VPC